src/libs/actions/Session/index.js

```diff
@@ -188,6 +189,7 @@ function createTemporaryLogin(authToken, email) {
         partnerUserSecret: autoGeneratedPassword,
         shouldRetry: false,
         forceNetworkRequest: true,
+        shouldProcessImmediately,
```

Comment:
Same here - I'm having trouble seeing how this is connected to the transition stuff.

Reply:
Removed this code and it doesn't seem to impact anything.